api/.prettierignore

```diff
@@ -0,0 +1,6 @@
+public
+dataOverlay
+Dockerfile
+package*
+repository
+svg
```

Comment:
I'm not quite sure if json files at `api/src/data` should be ignored by linter or not. These files are copied from the repo `data-files`. If we want to run prettier for these files as well, then the changes should be made in the `data-files`.

Reply:
Thanks for pointing that out! I think it sounds reasonable to add `src/data` to the ignore list. I can push a commit for that, if @ea agrees?

But speaking of which, I don't think the ignore list will be used when running prettier from the precommit hook. If we are bothered by that, we could add it as a separate issue.

Reply:
I just realized that we have `api/src/data/*.json` in the `.gitignore` file so I don't think this would cause any problems with the precommit hook since those files would never be picked up by git.

Reply:
Ah, good point@e0. Added in cf5058592f20f9fc84a3521481ace270ce0a545f